openshift_tooling/node_tuning_operator/node_tuning_operator.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env python
+
+import subprocess
+import time
+import sys
+
+#############################################################
+# Test: Node Tuning Operator: core functionality is working #
+#############################################################
+
+path_to_ssh_key = sys.argv[1]
+if len(path_to_ssh_key) == 0:
+    print ("Please run with path to ssh key. Ex. python node_tuning_operator.py /home/user/.ssh/libra.pem")
+    raise SystemExit
+
+# TODO - turn off colors on demand
+# Setting the output colors
+
+
+red = "\33[31m"
+on_red = "\33[41m"
+blue = "\33[34m"
+on_blue = "\33[44m"
+green = "\33[32"
+on_green = "\33[42m"
+reset = "\33[0m"
+
+
+# TODO - move print/execute steps to some new utils file of find what is in SVT already
+def print_title(title):
+    print("Running test: {}{}{}".format(on_blue, title, reset))
+
+
+def print_step(step_description):
+    print("\nStep: {}{}{}".format(on_blue, step_description, reset))
+
+
+def print_command(command_description):
+    print("Executing command: {}{}{}".format(blue, command_description, reset))
+
+
+def print_warning(warning):
+    print("{}{}{}".format(red, warning, reset))
+
+
+def passed(description):
+    print("{}STEP PASSED{}".format(on_green, reset))
+    if description is not None:
+        print("{}{}{}".format(green, description, reset))
+
+
+def fail(description):
+    print("{}STEP FAILED{}".format(on_red, reset))
+    if description is not None:
+        print("{}{}{}".format(red, description, reset))
+    raise SystemExit
+
+
+def execute_command(command_to_execute):
+    print_command(command_to_execute)
+    try:
+        value_to_return = subprocess.check_output(command_to_execute, shell=True)
+    except subprocess.CalledProcessError as exc:
+        value_to_return = exc.output
+    return value_to_return
+
+
+def execute_command_on_node(external_address, node_address, command_to_execute):
+    command_on_node = "ssh -i {} -o ProxyCommand='ssh -A -i {} -W %h:%p core@{}' core@{} {}".format(path_to_ssh_key, path_to_ssh_key, external_address, node_address, command_to_execute)
```

Comment:
Add `-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no` into ssh command to avoid ssh asking confirmations.

Reply:
Done, fixed